mohawk/base.py

```diff
@@ -77,7 +77,7 @@ def _authorize(self, mac_type, parsed_header, resource,
                                                ts=parsed_header['ts'],
                                                id=resource.credentials['id']))
         else:
-            log.warn('seen_nonce was None; not checking nonce. '
+            log.warning('seen_nonce was None; not checking nonce. '
                      'You may be vulnerable to replay attacks')
```

Comment:
Sorry for the nitpick but could you align the quotes with the opening parenthesis like this:
````py
log.warning('seen_nonce was None; not checking nonce. '
            'You may be vulnerable to replay attacks')
````